contracts/.changeset/config.json

```diff
@@ -1,16 +0,0 @@
-{
```

Comment:
Don't delete this

Reply:
🙏 

8ca267d940c0320a6d9a8fd235aa5f4c6cf0eb28